osidb/models/affect.py

```diff
@@ -158,6 +158,8 @@ class AffectFix(models.TextChoices):
         choices=NotAffectedJustification.choices, max_length=100, blank=True
     )
 
+    resolution_dt = models.DateTimeField(null=True, blank=True)
```

Comment:
This is pretty minor so feel free to ignore as that might be not wort addressing. I was just looking at other timestamps and see `created_dt`, `updated_dt`, `updated_until_dt`, `task_updated_dt`, `supported_*` `reported_dt`, `shipped_dt` so was thinking that `resolved_dt` would be more consistent with them (all past tense). However, I see we have inconsistencies already in `unembargo_dt`, `major_incident_start` so it does not matter much. Leaving it up to you.

Reply:
Makes sense, I've changed resolution_dt to resolved_dt